### ISSUE TITLE
Alternative to georaster dependency

### ISSUE THREAD

**Issue by perrygeo** [perrygeo/python-rasterstats#76]
Currently we have some [tight coupling](http://www.dotnet-stuff.com/tutorials/c-sharp/understanding-loose-coupling-and-tight-coupling) between the `GeoRaster` class and `zonal_stats`. I'd like to propose a change that would allow for looser coupling (as well as remove the dependencies on `scipy`, which slows down CI build times considerably, and `osgeo.osr`, which is not considered safe)

Instead of `zonal_stats(..., opt_georaster=True)` we could decouple them into:

```
zs = zonal_stats(...)
newstats = []
for s in zs:
    s['georaster'] = georasters.GeoRaster(
        s['mini_raster'], s['mini_raster_GT'], nodata_value=s['mini_raster_NDV'], projection=spatial_ref)
    newstats.append(s)
```
 the intention would be that `zonal_stats` just produces the data and the calling function could do whatever it needed: transform to a GeoRaster, write to GeoTiff, etc. That way when/if the GeoRaster API changes, I don't have to update it on this end or worry about supporting multiple versions. Does this sound OK to you @ozak ?

**Comment by ozak** [comment]
@perrygeo Sure, that sound perfectly fine. I was actually planning on something similar, tough I had had not time to do it. After the addition of the option to produce ``mini_rasters`` it seemed this would be a perfectly fine and simple way of having both modules interact.

**Comment by perrygeo** [comment]
resolved in 481783623c344c65221955dd560b089a0c6d6e1a
